src/scanning/Thumbnail/Thumbnail.jsx

```diff
@@ -0,0 +1,33 @@
+import "./Thumbnail.scss";
+import { getThumbnailAltAndLink, getThumbnailHeader } from "../scanning.js";
+
+export const Thumbnail = ({ name, ra, dec, url }) => {
+  const { alt, link } = getThumbnailAltAndLink(name, ra, dec);
+  return (
+    <div className={`thumbnail ${name}`}>
+      <div className="thumbnail-name">{getThumbnailHeader(name)}</div>
+      <div className="thumbnail-image">
+        <img
```

Comment:
Does ionic cache resources like the browser would? Just curious about whether or not these cutouts will be cached in the app, which is very nice UX-wise

Reply:
Ionic embeds the code inside a native webview it should have the same behavior as the browser